src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--

```

Comment:
Bunch of WS changes when I just tried to fix a spelling error. I can revert it if you like

Reply:
Yeah they would need to be reverted before merging unfortunately.

Reply:
This is one our old tooling problems. The resw files _must_ be saved with visual studio, otherwise they'll flip/flop their line endings and whitespace every commit that touches them. It's obnoxious.

Reply:
Will fix.

Reply:
Fixed.